augly/tests/image_tests/functional_unit_test.py

```diff
@@ -17,7 +17,10 @@ def test_blur(self):
 
     def test_brightness(self):
         self.evaluate_function(imaugs.brightness)
-
+        
+    def test_skew(self):
+        self.evaluate_function(imaugs.skew)
```

Comment:
@tanujdhiman please run these tests, add the output image file, & make sure they pass.

Reply:
Yes, I've added too.

Reply:
Coming to `requirements.txt` when I was running it in my local system then it will throw an error. (PIP is upgraded)

![image](https://user-images.githubusercontent.com/56601466/123770454-ee2e7880-d8e7-11eb-9669-ce39230dc733.png)